Docker: Razzle build 'Webpack' error

Frontend 'make build-image' fails with following build error. It seems webpack is not available in docker builds only. Yarn build successfully completes on local machine. There are no errors. 

```
$ yarn omelette && yarn patches
yarn run v1.22.17
$ ln -sf node_modules/@plone/volto/ omelette
Done in 0.04s.
yarn run v1.22.17
$ /bin/bash patches/patchit.sh > /dev/null 2>&1 ||true
Done in 0.04s.
Done in 238.86s.
Removing intermediate container e1d6cbd070f7
 ---> 37d81c8602db
Step 7/15 : COPY . .
 ---> f1f52f7ed646
Step 8/15 : RUN yarn build
 ---> Running in 20e6291a3aff
yarn run v1.22.17
$ razzle build
internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module 'webpack'
Require stack:
- /usr/src/app/node_modules/mini-css-extract-plugin/dist/index.js
- /usr/src/app/node_modules/mini-css-extract-plugin/dist/cjs.js
- /usr/src/app/node_modules/razzle/config/createConfigAsync.js
- /usr/src/app/node_modules/razzle/scripts/build.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18)
    at Object.<anonymous> (/usr/src/app/node_modules/mini-css-extract-plugin/dist/index.js:8:39)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/src/app/node_modules/mini-css-extract-plugin/dist/index.js',
    '/usr/src/app/node_modules/mini-css-extract-plugin/dist/cjs.js',
    '/usr/src/app/node_modules/razzle/config/createConfigAsync.js',
    '/usr/src/app/node_modules/razzle/scripts/build.js'
  ]
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
The command '/bin/sh -c yarn build' returned a non-zero code: 1

```

--- a/frontend/Dockerfile
+++ b/frontend/Dockerfile
@@ -13,7 +13,7 @@ RUN apt-get update \
 # A wildcard is used to ensure both package.json AND package-lock.json are copied
 # where available (npm@5+)
 COPY package.json yarn.lock  ./
-RUN yarn install --prod
+RUN yarn install
 
 # Bundle app source
 COPY . .

